File: Squad 01 - Grafos/Raíssa Rocha/atividade1.py
def soma(numero01, numero02):
    resultado = numero01 + numero02
    return resultado


while True:
    try:
        numero01 = int(input("Digite o primeiro número inteiro: "))
        break
    except ValueError:
        print("Entrada inválida! Por favor, digite apenas números inteiros.")

while True:
    try:
        numero02 = int(input("Digite o segundo número inteiro: "))
        break
    except ValueError:
        print("Entrada inválida! Por favor, digite apenas números inteiros.")

resultado = soma(numero01, numero02)

print(f"A soma dos números é {resultado}")

File: Squad 01 - Grafos/Gabriel Luhan/Atividade1.c
#include <stdio.h>

/*
 * Parâmetros:
 *   a: primeiro valor
 *   b: segundo valor
 *
 * Retorno:
 *   a soma de a e b
 */
int soma(int a, int b) {
    int resultado = a + b;
    return resultado;
}

int main(void) {
    int a, b;

    /* Lê os dados de entrada */
    scanf("%d", &a);
    scanf("%d", &b);

    /* Resultado */
    printf("%d\n", soma(a, b));

    return 0;
}